migrations/tests/test_0010_add_sequential_number_to_assignment.py

```diff
@@ -0,0 +1,92 @@
+def test_migration(write, finalize, assert_model):
+    write({"type": "create", "fqid": "meeting/1", "fields": {"id": 1}})
+    write(
+        {"type": "create", "fqid": "assignment/1", "fields": {"id": 1, "meeting_id": 1}}
+    )
+    write(
+        {"type": "create", "fqid": "assignment/2", "fields": {"id": 2, "meeting_id": 1}}
+    )
+
+    finalize("0010_add_sequential_number_to_assignment")
```

Comment:
Since the migration is more complex, we should test that it works with the two-stage migration. You should be able to create a `migrate` fixture analogous to the `finalize` one (see https://github.com/OpenSlides/openslides-backend/blob/9fe6aad0c7f61e2b9b9ef7d7a2c682b281d2f6ba/migrations/tests/conftest.py#L49). Use it here, write another event after that und then do a `finalize`. Assert that the sequential numbers are correct.

Also add another meeting to assert that the migration works correctly for multiple meetings.

Reply:
Done.